joomla/joomla-cms
# [4.0] [tpl_protostar] Delete pagination.php in protostar template (deprecated)

### Summary of Changes
This PR just removes the pagination.php for the new installations because this method to override the pagination is deprecated and might be confusing.

The new place to override this is in the path layouts/joomla/pagination/


### Testing Instructions

Apply patch and see if still all paginations are shown correctly.

![image](https://cloud.githubusercontent.com/assets/828371/23230805/98fb72ea-f945-11e6-8208-27cf56291f81.png)
